### ISSUE TITLE
a fun story about how a google product manager contributed to our class

### ISSUE THREAD

**Issue by mikeizbicki** [mikeizbicki/cmc-csci181#4]
I have a fun story to share with everyone.

@GalOshri is a product manager for google working on tensorboard.  He found out about our deep learning class by browsing github, and wanted to share with us some new features of tensorboard that will be helpful for our new online class format.  This is one of the great advantages of open source collaboration on platforms like github---you never know who is going to contact you about your projects and offer cool suggestions.  Also, as we switch to this online format, I hope this story will help motivate you to master these new tools because they really are widely used in practice for remote collaboration.

@GalOshri's suggestion is that we use the website https://tensorboard.dev to host our tensorboard log files.  This is a new service provided by google to help online collaboration for deep learning.  Essentially, this website shows exactly the same results as running tensorboard on your local computer, but the results are now accessible by anyone and not just you.  

I'll have instructions in our next homework (hw7) about we'll use this service, but in the meantime you can see an example at: https://tensorboard.dev/experiment/9eiEYkDMSO68SwDSEoc4Gg/#scalars&_smoothingWeight=0.99 .  In the example, I have 4 different training runs for hw6 each with a different combination of batch size and learning rate.  Notice that the red line is converging much faster than the other lines because the larger batch size lets us use a larger learning rate.  I created this tensorboard output in response to question #3, and I'll be creating similar outputs to help answer future questions.
